Formula/o/open-mpi.rb

```diff
@@ -38,6 +39,10 @@ def install
     if OS.mac?
       # Otherwise libmpi_usempi_ignore_tkr gets built as a static library
       ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
+
+      # Work around asm incompatibility with new linker (FB13194320)
+      # https://github.com/open-mpi/ompi/issues/11935
+      ENV.append "LDFLAGS", "-Wl,-ld_classic" if DevelopmentTools.clang_build_version >= 1500
```

Comment:
I think it needs to be `FCFLAGS` instead (or in addition?)

Reply:
Not sure if open-mpi reads wrapper stuff from the environment or not though so might need a hack to insert.

Reply:
Basically: whatever is required to make it appear in `share/openmpi/mpifort-wrapper-data.txt`. I edited that file locally and it seems to work.